java/src/jmri/jmrit/beantable/LogixTableAction.java

```diff
@@ -98,12 +110,19 @@
  * lists:<br>
  * 1. the previous policy - Trigger on change of state only <br>
  * 2. the new default - Trigger on any enabled state calculation
- * <p>
  * Jan 15, 2011 - Pete Cressman
- *
+ * <p>
```

Comment:
Just to be clear, I should add the @since tag to each new method?  Or just the class text, or both?